docs/source/concoreexample.rst

```diff
@@ -0,0 +1,46 @@
+Concore examples
```

Comment:
This should be named as something else, and not "Concore examples."

Perhaps, "The Contribute Action?"